### PR TITLE
Rename "Unthrottle" to "Fast-forward"

### PR DESCRIPTION
**Pull request by hrydgard**
"Unthrottle", while kinda correct, is not the most obvious name for a feature that just removes the speed limit and lets the emulator run as fast as it can.

So, change it to fast-forward, which seems a lot more intuitive.

### CODE REVIEW

File: Core/HW/StereoResampler.cpp [hunk 1]
```diff
@@ -266,15 +266,15 @@ void StereoResampler::PushSamples(const s32 *samples, unsigned int numSamples) {
 	u32 indexW = m_indexW.load();
 
 	u32 cap = m_maxBufsize * 2;
-	// If unthrottling, no need to fill up the entire buffer, just screws up timing after releasing unthrottle.
-	if (PSP_CoreParameter().unthrottle) {
+	// If unthrottling, no need to fill up the entire buffer, just screws up timing after releasing the fast-forward button.
```
Comment (unknownbrackets) [inline Core/HW/StereoResampler.cpp:269]: nit: If we're trying to change everywhere, there's still "unthrottling" here.

-[Unknown]

**Review by unknownbrackets — APPROVED** [review]

File: UI/GamepadEmu.h [hunk 2]
```diff
@@ -264,7 +264,7 @@ namespace CustomKey {
 		{ "L", ImageID("I_L"), CTRL_LTRIGGER },
 		{ "R", ImageID("I_R"), CTRL_RTRIGGER },
 		{ "RapidFire", ImageID::invalid(), VIRTKEY_RAPID_FIRE },
-		{ "Unthrottle", ImageID::invalid(), VIRTKEY_UNTHROTTLE },
+		{ "FastForward", ImageID::invalid(), VIRTKEY_FASTFORWARD },
```
Comment (iota97) [inline UI/GamepadEmu.h:267]: The string should be "Fast-forward" so it get translated.

File: assets/lang/it_IT.ini [hunk 3]
```diff
@@ -653,7 +653,7 @@ Texture Replacement = Sostituzione delle texture
 Toggle Fullscreen = Passa a Schermo Intero
 Toggle mode = Modalità Scambio
 Triangle = Triangolo
-Unthrottle = Non velocizzare
+Fast-forward = Non velocizzare
```
Comment (iota97) [inline assets/lang/it_IT.ini:656]: Should be `Fast-forward = Velocizza`, as this is literally "Don't speed up"

File: assets/lang/pt_BR.ini [hunk 4]
```diff
@@ -677,7 +677,7 @@ Texture Replacement = Substituição das texturas
 Toggle Fullscreen = Alternar pra Tela cheia
 Toggle mode = Alternar modo
 Triangle = Triângulo
-Unthrottle = Desacelerador
+Fast-forward = Desacelerador
```
Comment (iota97) [inline assets/lang/pt_BR.ini:680]: I don't speak portuguese/brasialian myself but this seem kinda the same mistake of italian translation, might worth a check if you know someone that speak it.

**Comment by iota97** [comment]
Also seem that ffmpeg got in the way again :)

File: assets/lang/it_IT.ini (hunk 3 above)
Comment (hrydgard) [reply assets/lang/it_IT.ini:656]: Yeah, it seems Unthrottle is just very confusing and several languages had opposite translations.

I've fixed them using Google Translate, hopefully they'll at least be in the right direction...

**Comment by hrydgard** [comment]
haha, thanks iota97. finally did a git submodule update, shouldn't happen again :P

**Review by Ams22ttl — APPROVED** [review]

**Pull request merged**